### DISCUSSION TITLE
How do I merge my master branch into my github page?

### DISCUSSION THREAD

**Discussion by MehmetOmercik** [community/community#35916] [Other Feature Feedback, Questions, & Ideas]
So I am trying to update my GitHub page that contains my CV but for some reason, it isn't updating and states that I need to "Merge branch 'master' of [my website]" even though I have no other branches apart from the master branch. I'm trying to push it back through again and merge commands but nothing is working. When I look further into it, it states that there is an issue with Jekyll and not GitHub itself as I can see that the file is updated. 

Jekyll states:
GitHub-pages 227 | Error:  No such file or directory @ dir_chdir - /github/workspace/docs" 

I don't have any idea what this means tbh and I would appreciate some help.

![image](https://user-images.githubusercontent.com/111444136/195377517-250329e3-5d77-4d6e-ae26-dcf16c8da98f.png)

**Comment by airtower-luna** [comment]
Your repository seems to be private so I can't check, but most likely it is configured to deploy from a `docs/` subdirectory which you don't have. I assume you'll want to deploy from the root of the repository instead, check the Pages settings.

**Comment by MehmetOmercik** [comment]
Right so I tried that and now I still have the same problem and I have a 404 error on the website
